python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -138,6 +138,7 @@ public JSONObject provideResourceSchemaJSONObject() {
             .logicalResourceIdentifier(request.getRequestData().getLogicalResourceId())
             .nextToken(request.getNextToken())
             .region(request.getRegion())
+            .snapshotRequested(request.getSnapshotRequested())
```

Comment:
We should not add change in the generated code here. This will cause build break if only upgrade the PyPi java plugin.

The better place like this https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/src/main/java/software/amazon/cloudformation/LambdaWrapper.java#L275

We won't even need tool release.